CMakeLists.txt

```diff
@@ -136,11 +134,8 @@ list(APPEND EXTRA_LIBRARIES ${CMAKE_DL_LIBS})
 # add_definitions( -DHAVE_CONFIG_H )
 
 if (EXTERNAL_BOOST)
-  set(NEEDED Boost_VERSION 1.55.0)
-  if(CMAKE_CXX_COMPILER_ID MATCHES "GNU" AND NOT CMAKE_CXX_COMPILER_VERSION VERSION_LESS "5")
-    set(NEEDED_Boost_VERSION 1.58.0) #issue #8
-  endif()
-  find_package(Boost ${NEEDED_Boost_VERSION} COMPONENTS mpi python serialization system filesystem)
+  set(NEEDED Boost_VERSION 1.61.0)
+  find_package(Boost ${NEEDED_Boost_VERSION} REQUIRED COMPONENTS mpi python serialization system filesystem)
```

Comment:
More fun, this ${NEEDED_Boost_VERSION} is not a valid name; I thought that NEEDED is some special keyword for set() but apparently not.
When I set it explicitly like you suggest then FindBoost complained because on travis, version 1.58 is only available, with the set() there was effectively no requirements on the Boost version.